pkg/chart/chart.go

```diff
@@ -58,3 +60,30 @@ func updateContainerImageRegistry(valuesFile string, targetRepo *api.TargetRepo)
 	}
 	return errors.Trace(err)
 }
+
+// updateReadmeFile performs some substitutions to a given README.md file.
+func updateReadmeFile(readmeFile, sourceURL, targetURL, chartName, repoName string) error {
+	klog.V(2).Infof("Updating README file")
+	readme, err := ioutil.ReadFile(readmeFile)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	// Update helm repo add with string replacement
+	addBitnamiRepoLine := fmt.Sprintf("helm repo add bitnami %s", sourceURL)
+	addCustomRepoLine := fmt.Sprintf("helm repo add %s %s", repoName, targetURL)
+	newContent := strings.ReplaceAll(string(readme), addBitnamiRepoLine, addCustomRepoLine)
+	// Update bitnami/chart references with regex
+	regexString := fmt.Sprintf(`(?m)(\s)(bitnami/%s)(\s)`, chartName)
+	regex := regexp.MustCompile(regexString)
```

Comment:
declare the regex expression as a top-level package variable so it fails at initialization time, not once the program reaches this point

Reply:
note that for this case the regex depends on the chartName

Reply:
then please consider using [`Compile`](https://golang.org/pkg/regexp/#Compile) rather than [`MustCompile`](https://golang.org/pkg/regexp/#MustCompile) since the former is thought to be used in this kind of case